crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -147,3 +172,18 @@ where
             .collect()
     }
 }
+
+impl<TYPES, PUBKEY: SignatureKey + 'static> GeneralStaticCommittee<TYPES, PUBKEY>
+where
+    TYPES: NodeType<SignatureKey = PUBKEY, ElectionConfigType = StaticElectionConfig>,
+{
+    /// get the non-staked builder nodes
+    pub fn non_staked_nodes_count(&self) -> usize {
+        self.committee_nodes_without_stake.as_ref().unwrap().len()
```

Comment:
Same question as above. In this case, it seems like a good default would be `0` perhaps, and if you decided to remove the `Option` from the vec above it would give a clean fail case. If we decide to keep this method, it might be worth documenting that this function can panic.

Reply:
Looks like this message got put here out of order. Please see the below `is it safe to unwrap here?` comment that I am referencing when I say "above"